### DISCUSSION TITLE
License requirements for distribution

### DISCUSSION THREAD

**Discussion by dhirschfeld** [tonybaloney/Pyjion#479] [General]
Hi @tonybaloney :wave:
I'd like to be able to install `pyjion` via `conda` so I've opened a (draft) PR to do so in `conda-forge`:
* https://github.com/conda-forge/staged-recipes/pull/17937

I assume the `dotnet` submodule is used to build this package so wanted to check if that means we have to include the .NET license in any redistribution of `pyjion`?

**Comment by dhirschfeld** [comment]
Also, if you're interested in being added as a maintainer of the `pyjion` package on `conda-forge` I'm happy to add you - just let me know.

**Reply by tonybaloney** [reply]
Awesome. .NET 6 is MIT, same as Pyjion, so all good to distribute. Checkout the Dockerfile or the Manylinux workflow in this repo for tricks on packaging. 
Please add me to the recipe as well, I'm already a member of conda-forge

**Comment by dhirschfeld** [comment]
I got the linux build working and passing tests so I think the `staged-recipes` PR is ready to merge :tada:

As I'm only initially interested in the linux build I'm happy for the other platforms to be enabled on the feedstock itself as and when someone finds the time/motivation.

https://github.com/conda-forge/staged-recipes/pull/17937
